### PR TITLE
Add simple pub sub load test

### PR DESCRIPTION
**Pull request by dilanSachi**
## Purpose
$subject
Fixes https://github.com/ballerina-platform/ballerina-standard-library/issues/4738
## Checklist
- [x] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests
- [ ] Updated the spec
- [ ] Checked native-image compatibility

### CODE REVIEW

File: load-tests/simple_publisher_subscriber/src/main.bal [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+    //     log:printInfo("Listener started.");
+    //     errorCount = 0;
+    //     sentCount = 0;
+    //     receivedCount = 0;
+    //     startedTime = time:utcNow();
+    //     endedTime = time:utcNow();
+    //     finished = false;
+    //     _ = start publishMessages();
+    //     log:printInfo("Started publishing messages.");
+    // }
```
Comment (ayeshLK) [inline load-tests/simple_publisher_subscriber/src/main.bal:67]: Shall we remove the commented lines ?

File: load-tests/simple_publisher_subscriber/src/main.bal [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+
+service /mqtt on new http:Listener(9100) {
+
+    resource function get publish() returns boolean {
+        log:printInfo("Received request to start publishing messages.");
+        error? result = startListener();
+        log:printInfo("Started listener.");
+        if result is error {
+            return false;
+        }
```
Comment (ayeshLK) [inline load-tests/simple_publisher_subscriber/src/main.bal:57]:
```suggestion
        error? result = startListener();
         if result is error {
            return false;
        }
        log:printInfo("Started listener.");
```

Comment (dilanSachi) [reply load-tests/simple_publisher_subscriber/src/main.bal]: Updated

**Review by ayeshLK — APPROVED** [review]
LGTM

**Pull request merged**